rdb/src/rdb/core.clj

```diff
@@ -0,0 +1,135 @@
+(ns rdb.core
+  (:require [clojure [pprint :refer :all]
+                     [string :as str]]
+            [clojure.java.io :as io]
+            [clojure.tools.logging :refer [debug info warn]]
+            [jepsen [core      :as jepsen]
+                    [db        :as db]
+                    [util      :as util :refer [meh timeout]]
+                    [control   :as c :refer [|]]
+                    [client    :as client]
+                    [checker   :as checker]
+                    [model     :as model]
+                    [generator :as gen]
+                    [nemesis   :as nemesis]
+                    [store     :as store]
+                    [report    :as report]
+                    [tests     :as tests]]
+            [jepsen.control [net :as net]
+                            [util :as net/util]]
+            [jepsen.os.debian :as debian]
+            [jepsen.checker.timeline :as timeline]
+            [rethinkdb.core :refer [connect close]]
+            [rethinkdb.query :as r]
+            [knossos.core :as knossos]
+            [cheshire.core :as json])
+  (:import (clojure.lang ExceptionInfo)))
+
+(defn copy-from-home [file]
+    (c/scp* (str (System/getProperty "user.home") "/" file) "/root"))
+
+(defn db []
+  "Rethinkdb (ignores version)."
+  (reify db/DB
+
+    (setup! [_ test node]
+      (debian/install [:libprotobuf7 :libicu48 :psmisc])
+      (info node "Starting...")
+      ;; TODO: detect server failing to start.
+      (c/ssh* {:cmd "killall rethinkdb bash"})
+      (c/ssh* {:cmd "
+# Add a 100ms delay to help find edge cases.
```

Comment:
What's your policy on code that's sometimes useful when fiddling but isn't being actively used?  Should I just cut these commented-out lines?